### ISSUE TITLE
Server returned HTTP response code: 400 for URL: http://api.skype.com/users/batch/profiles

### ISSUE THREAD

**Issue by MrPowerGamerBR** [Delthas/JavaSkype#11]
```
java.io.IOException: Server returned HTTP response code: 400 for URL: http://api.skype.com/users/batch/profiles
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1840)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1441)
	at fr.delthas.skype.WebConnector.sendRequestJson(WebConnector.java:260)
	at fr.delthas.skype.WebConnector.updateContacts(WebConnector.java:115)
	at fr.delthas.skype.WebConnector.start(WebConnector.java:44)
	at fr.delthas.skype.Skype.connect(Skype.java:92)
	at fr.delthas.skype.Skype.connect(Skype.java:68)
	at com.mrpowergamerbr.wow.Testing.main(Testing.java:12)
```
All contacts are loaded, but all of them are shown as "OFFLINE" and doesn't have any mood.

**Comment by MrPowerGamerBR** [comment]
Also, this doesn't happen on my bot account (which has 41 contacts) but happens with my main account (with ~730 contacts)

**Comment by Delthas** [comment]
Normally this should be fixed now (1.0.10). The reason was I was sending a request to search for contacts but the request would get too long if you had too many contacts.

**Comment by MrPowerGamerBR** [comment]
@Delthas fixed in the latest version, thanks!
